std_msgs/Header header

# flattened 4x4 matrix representing the camera extrinsics
float64[16] M_CL
